tw-install.sh

```diff
@@ -49,8 +49,29 @@ yes | sudo apt install uuid-dev
 yes | sudo apt install cmake
 yes | sudo apt install gnutls-bin
 
-# install taskwarrior and add first task
-yes | sudo apt install taskwarrior
+#Install Taskwarrior
+VERSION=latest
+
+while true; do    
+    echo -e "Select the version of taskwarrior to install: \n"
+    cat taskwarrior.versions
+    echo -e "Choose from above list ( 1-12 )"
+    read ver
+    echo ""
+    case $ver in
+        [1-9]|[1-1][0-2] ) VERSION=$(cat taskwarrior.versions|grep "$ver)" | awk '{print $2}'|head -1) && break;;
```

Comment:
Awesome work! Thank you!
I do not exactly know what the  
```
[1-9]|[1-1][0-2] )
```
does, would it perhaps be possible to include a comment to briefly describe what this line does/how it works?

Reply:
@a-t-0  It's regex. Basically, it is only allowing options from 1 to 12. If the user chooses other than this script will ask again for versions.

Reply:
@Rahul-Soni28  Thank you for your elaboration, and my apologies for my vagueness. I meant a comment inside the code, (instead of a comment in this thread). That way, if people build upon your work in this project, without having that specific knowledge, they can easily get a better understanding of what the code does.

Reply:
@a-t-0  It's done.  I have added some comments. Now it makes more sense.